yt_dlp/extractor/eplus.py

```diff
@@ -0,0 +1,110 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    try_call,
+    unified_timestamp,
+)
+
+
+class EplusIbIE(InfoExtractor):
+    IE_NAME = 'eplus:inbound'
+    IE_DESC = 'e+ (イープラス) overseas'
+    _VALID_URL = r'https?://live\.eplus\.jp/ex/player\?ib=(?P<id>([A-Za-z\d]|%2B|%2F){86}(%3D){2})'
+    _TESTS = [{
```

Comment:
Why are all tests "fake"? Use a real test that currently works for you locally and then add `'skip': 'live'`

Reply:
Well, the reason is:

> All usable URLs are from purchased events. Posting a real URL might give eplus a chance to find users who shared their tickets.

Reply:
Sorry, forgot about that